packages/editor/src/components/post-featured-image/index.js

```diff
@@ -202,17 +201,15 @@ function PostFeaturedImage( {
 								{ !! featuredImageId && (
 									<HStack className="editor-post-featured-image__actions">
 										<Button
-											// TODO: Switch to `true` (40px size) if possible
-											__next40pxDefaultSize={ false }
+											__next40pxDefaultSize
```

Comment:
<table>
<tr>
<td>Before</td>
<td>After</td>
</tr>
  <tr>
    <td>
<img width="280" alt="replace-remove before" src="https://github.com/user-attachments/assets/348286d7-0a4a-406a-97f1-5aa74a0155dc">

</td>
    <td>
<img width="280" alt="replace remove after" src="https://github.com/user-attachments/assets/9f42e00d-2c15-4170-a1f6-8ce700fa799a">

</td>
  </tr>
</table>

Reply:
Looks like the before/after images are reversed in this one.

We probably need an `!important` here:

https://github.com/WordPress/gutenberg/blob/9523fb4a6d568f34a217c9cab45f29413c4904f7/packages/editor/src/components/post-featured-image/style.scss#L56

Reply:
<table>
<tr>
<td>Before</td>
<td>After</td>
</tr>
  <tr>
    <td>
<img width="280" alt="Screenshot 2024-09-12 at 12 57 25 AM" src="https://github.com/user-attachments/assets/00f25c13-8ad6-4bb6-8daa-a51800f5ec89">


</td>
    <td>
<img width="279" alt="Screenshot 2024-09-12 at 12 55 49 AM" src="https://github.com/user-attachments/assets/3511ce68-8b7d-4844-965c-568093519a3d">

</td>
  </tr>
</table>

Hii @mirka , `height: auto !important`; added for featured image. 
PFA new screenshots.

Reply:
I noticed a weird flash on focus, but this happens on trunk too so I'll post a separate issue.

https://github.com/user-attachments/assets/6da090d2-002b-4ac3-b923-56bb59050244

Reply:
Issue submitted at #65299